app/src/main/res/values/strings.xml

```diff
@@ -77,4 +77,4 @@
     <string name="contextmenu_app_open_settings">Settings</string>
     <string name="contextmenu_folder_app_delete">Uninstall</string>
     <string name="app_disabled_suffix">(Disabled)</string>
-</resources>
\ No newline at end of file
+</resources>
```

Comment:
I edited this file using github web editor (I am not at pc rn), this line was edited automatically by it. Was not present in preview.